compiler/src/AsmOutput.sk

```diff
@@ -2012,16 +2012,11 @@ mutable class .AsmOutput{
     asm.print("i8* bitcast (%N to i8*)" % id)
   }
 
-  // As in the runtime, strings always take a multiple of 8 bytes
-  // and strings >= cstr_buffer_size must have a trailing '\0'.
+  // Static strings always take a multiple of 8 bytes and have a trailing '\0'
   private readonly fun paddedStringSize(s: UTF8String): Int {
     size = s.utf8.size();
 
-    // WARNING! This must equal CSTR_BUFFER_SIZE in the runtime!
-    cstr_buffer_size = (80 - 8 + 1);
-
-    // Guarantee zero-padding for long enough strings.
-    if (size >= cstr_buffer_size && s.utf8[size - 1] != 0) {
+    if (size == 0 || s.utf8.last() != 0) {
```

Comment:
Why don't we just remove the condition so it's easy to know how much padding is added?

Reply:
Yes, removing is ok. This test is in practice always true anyhow, since no static strings that contain an embedded nul at the end are ever created.